Elaborate `getting started` section of README

ixxie opened the pull request:
Distinguish  installation, (minimal) configuration and asset handling steps. Add details for getting the plugin working in SvelteKit.

donmccurdy commented on README.md:
```diff
@@ -28,16 +30,64 @@ Vite:
+
+### 4. Execution on Build
+
+The plugin will optimize assets _at build time_. This means you will not observe results when running `npm run dev`. To test the optimization, run `npm run build` and `npm run preview`.
```
Nit: the current wording is specific to Vite, npm clients, and particular choices of macro names. Maybe we can clarify it as —

```suggestion
The plugin will optimize assets _at build time_. Vite has [several modes](https://vitejs.dev/guide/env-and-mode.html#modes), and optimization occurs in production (`vite build`) and preview (`vite preview`) but not development (`vite`) modes.
```

donmccurdy reviewed (commented):
a couple suggestions, but this looks great – thank you!

ixxie commented:
@donmccurdy thanks for the suggestions... of course my knowledge is limited here so its nice you filled in the gaps.

I squashed everything together and added you as a coauthor in the commit message.

juniorxsound reviewed (approved):
LGTM - thanks @ixxie! @donmccurdy thoughts?

donmccurdy commented on README.md:
```diff
@@ -28,16 +30,68 @@ Vite:
+
+See [advanced configuration](#advanced-configuration) for more complex examples.
+
+_SvelteKit:_ it seems that in SvelteKit SSR should be disabled in order to avoid [mysterious `ReferenceError` issues](https://github.com/nytimes/rd-bundler-3d-plugins/issues/14#issuecomment-1410620469).
```
Would you mind filing an issue (on this repo) about SvelteKit SSR too? Does seem very weird that our plugin would affect that. 🫤

ixxie replied:
I created the issues and referenced them in the README:

https://github.com/sveltejs/kit/issues/9000
https://github.com/nytimes/rd-bundler-3d-plugins/issues/19

donmccurdy reviewed (approved):
One optional suggestion / looks good to me!
